### ISSUE TITLE
Safari and Microsoft explorer does not work

### ISSUE THREAD

**Issue by MosheLavee** [ajwecker/TS1218#69]
Just got a message from a user that he cannot open the project on Safari. Same on explorer in my laptop...

**Comment by dstoekl** [comment]
Safari problem (screen 16/9).  Safari is VERY important. Is this only a resolution issue or a deeper one? What can be done?
![image](https://user-images.githubusercontent.com/39535215/52640157-1e6f4b00-2ed6-11e9-851e-e48ed10a63f3.png)

**Comment by dstoekl** [comment]
Indeed, on my ipad I can open the site. Excruciatingly slow (maybe due to connection, but we talk about 15 min or so). Most importantly: The essential button is invisible. The images of page and line are not loaded but covered by the logos. There should be a routine to render the logos invisible after 2 or 5 seconds  when the screen is small.

**Comment by MosheLavee** [comment]
@TsvikaKuflik @drore @ajwecker I guess we should not touch this now, since we should be now on the "feature freeze state". 
We might try to do a round of such fixes without deploying and upload a new version in a week or so.

**Comment by dstoekl** [comment]
I consider this to be the most important bug to solve. In France 20% of all users use Safari, it is the second after Chrome. I assume that many many users go to 929 via their ipads or cellphones. In France, Mac is extremely widespread. http://gs.statcounter.com/browser-market-share/all/france

**Comment by dstoekl** [comment]
In Israel it is 10% in the US even 30%
